### ISSUE TITLE
Switch docs CSS units from `rem` to `px`?

### ISSUE THREAD

**Issue by atomiks** [atomiks/30-seconds-of-css#85]
I've long used `rem` but when the window is being resized or elements are transitioning in height, it can cause ugly 1px jitters everywhere. With `px`, this doesn't occur.

`px` is also responsive on most browsers when the zoom level is changed.

https://hackernoon.com/rems-and-ems-and-why-you-probably-dont-need-them-664b9ce1e09f

@Chalarangelo

**Comment by Chalarangelo** [comment]
In general `rem` and `em` are a lot better than using `px`. It's softcoding vs hardcoding. I am strongly against using pure `px` everywhere. My usual approach is to do the following:
- Use `px` at the root element and elements that need to be specifically in `px` values (border thickness comes to mind usually).
- Use `rem` for pretty much everything else, so that the styling is uniform and can be easily changed based on the root element.
- Use `em` for things like bold and italics text, which is context-dependent.

**Comment by fejes713** [comment]
I totally agree with @Chalarangelo on that one. I've also read a lot about ems and rems and they're just fine in most cases. I don't think we should change it.

**Comment by Physsi07** [comment]
Hey, I would like to know if you want to change everything from rem to px, I can do this if you still want?

**Comment by skatcat31** [comment]
https://developer.mozilla.org/en-US/docs/Web/CSS/length

Might shed some light for you guys on how font sizes work

**Comment by atomiks** [comment]
The reason I've used `rem` is so everything scales down responsively when I use a root font size of `0.935rem` for <768px which fits more content on the screen. But there are rounding problems with `rem`, see:

<img src="https://i.gyazo.com/2b2bd14979ba26de611ec1153189d2a5.png" width="400">

That's why I just use `px` now for everything. Pretty sure it's fine for accessibility because `px` scales responsively with browser zoom levels / settings
